rfcs/text/0000-distributing-process-results-for-managed-access.md

```diff
@@ -0,0 +1,142 @@
+
+**Definitions**
+
+**Managed access data** Raw and processed genomic and phenotypic data from individuals whose consent agreements authorise data release only for specific research use to bona fide researchers.
+
+**Open access data** Raw and processed genomic and phenotypic data from individuals whose consent agreement authorise unrestricted sharing and summary level data generated from both open and managed access raw and processed genomic data.
+
+**European Genome Phenome Archive (EGA)** [EMBL-EBI archive](https://www.ebi.ac.uk/ega/) established to support the distribution of managed access data, providing data contributors a secure storage mechanism for their genomic data and a toolset which enables data contributors to manage access rights to their own data.
+
+**HCA Data Contributor Agreement** an agreement entered into between the Data Contributor and the HCA, prior to contribution and upload of data to the HCA DCP.  
+
+## User Stories
+
+As a **Researcher with a keyboard**, I want to find data that is similar to my own.  I download an expression matrix for that data, then analyze/visualize it locally using my own workflow. I would like this to work irrespective of the access permissions for the raw data and alignments for the data.
+
+As a **Researcher with a pipette**, I want to use the Data Browser and/or tertiary analysis portals to look for interesting patterns of a gene or cell type expression.  For example, I might look for all the tissues where my gene of interest is expressed, or try to find a combination of gene expressions to identify a pathway in my disease of interest.
+
+As a **Researcher with a pipette**, I would like to be able to share all my cellular resolution, human genomic data with the HCA DCP, including data which needs to be distributed via managed access as well as data which can be openly released. I want this to support the aim to build an atlas of all human cells. 
+
+## Scientific "guardrails" 
+
+This plan will be reviewed by members of the [Compliance working group]([https://github.com/HumanCellAtlas/dcp-community/blob/master/charters/Compliance-WG/charter.md](https://github.com/HumanCellAtlas/dcp-community/blob/master/charters/Compliance-WG/charter.md)) who must ensure the proposed design meets their guidelines for responsible handling of managed access genomic data.
+
+## Detailed Design
+
+The proposed data flow for managed access data takes some significant departures from the standard DCP data flow. 
+
+1. The DCP will not redistribute genomic data files which require managed access (fastq and bam).
+2. Genomic data files are still submitted via the ingestion service and HCA data and metadata validation but then brokered to the European Genome-Phenome Archive (EGA) rather than being exported to the Data Store. The EGA has long-standing experience in storing and distributing datasets which need managed rather than open access support which the DCP can leverage. 
```

Comment:
What about genomic data that is already in EGA?

There will be DACs that will require the DCP to be granted access to submit data to the EGA.   How are we going to mange getting approval from a larger number of DACs with different rules?